fs/fstest/file.go

```diff
@@ -0,0 +1,109 @@
+package fstest
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/stevvooe/continuity/sysx"
+)
+
+// Applier applies single file changes
+type Applier func(root string) error
```

Comment:
Let's do this as an interface and declare `ApplierFunc` as a type that calls itself.